src/flows/pipes/Notification/HTTP.tsx

```diff
@@ -0,0 +1,138 @@
+import React, {FC, useContext} from 'react'
+import {
+  Form,
+  FlexBox,
+  Input,
+  InputType,
+  ComponentSize,
+  SelectGroup,
+} from '@influxdata/clockface'
+
+import {PipeContext} from 'src/flows/context/pipe'
+
+const HTTP: FC = () => {
+  const {data, update} = useContext(PipeContext)
+
+  const updater = (field, value) => {
+    update({
+      endpointData: {
+        ...data.endpointData,
+        [field]: value,
+      },
+    })
+  }
+
+  const updateUrl = evt => {
+    updater('url', evt.target.value)
+  }
+
+  const updateAuth = (auth: string) => {
+    updater('auth', auth)
+  }
+
+  const updateUsername = evt => {
+    updater('username', evt.target.value)
+  }
+
+  const updatePassword = evt => {
+    updater('password', evt.target.value)
+  }
+
+  const updateToken = evt => {
+    updater('token', evt.target.value)
+  }
```

Comment:
<a href="https://imgflip.com/i/5cwsh7"><img src="https://i.imgflip.com/5cwsh7.jpg" title="made at imgflip.com"/></a><div><a href="https://imgflip.com/memegenerator">from Imgflip Meme Generator</a></div>

Reply:
i dont think we need to event "the user typed a character into an input" for every character. not a lot of value in that one

Reply:
But should we add events to this PR in general?

Reply:
definitely the ones that were removed
imma add one for changing the threshold column and condition, and one for changing the endpoint type
any more in mind?